src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    {
+      person.age && (
+      <p className="Person__age">{`I am ${person.age}`}</p>
+      )
+    }
+    {
+      person.isMarried !== true ? (
+        <p className="Person__partner">I am not married</p>
+      ) : (
+        <p className="Person__partner">
+          {`${person.partnerName} is my ${
+            person.sex === 'm' ? ('wife') : ('husband')
```

Comment:
for this check you can create variable
```
const partner = sex === 'm' ? 'wife' : 'husband';
```

there is no need to use `()` brackets here for ` 'wife'` or `'husband'`